### PR TITLE
Fix the Stackdriver Trace exporter import path

### LINKED ISSUE #289: exporter/trace/zipkin: fix the broken example
```
import "go.opencensus.io/trace/adaptor/stackdriver"

exporter, err := stackdriver.NewExporter(stackdriver.Options{ProjectID: *project})
if err != nil {
	log.Println(err)
} else {
	trace.RegisterExporter(exporter)
}
```

The import path from the sample is invalid.

### PR DESCRIPTION
**Pull request by rakyll**
Fixes #289.

### CODE REVIEW

File: exporter/trace/stackdriver/stackdriver.go [hunk 1]
```diff
@@ -16,14 +16,13 @@
 //
 // Example:
 //
-// 	import "go.opencensus.io/trace/adaptor/stackdriver"
+// 	import "go.opencensus.io/exporter/trace/stackdriver"
```
Comment (odeke-em) [inline exporter/trace/stackdriver/stackdriver.go]: Also please add "go.opencensus.io/trace" lest the example still hangs so
```go
//  import (
//        "go.opencensus.io/exporter/trace/stackdriver"
//        "go.opencensus.io/trace"
//  )
```

Comment (rakyll) [reply exporter/trace/stackdriver/stackdriver.go]: Done, thanks.

File: exporter/trace/stackdriver/stackdriver.go [hunk 2]
```diff
@@ -16,14 +16,16 @@
 //
 // Example:
 //
-// 	import "go.opencensus.io/trace/adaptor/stackdriver"
+// 	import (
+//        "go.opencensus.io/exporter/trace/stackdriver"
+//        "go.opencensus.io/trace"
+//  )
```
Comment (odeke-em) [inline exporter/trace/stackdriver/stackdriver.go]: Indent this please :)

Comment (rakyll) [reply exporter/trace/stackdriver/stackdriver.go]: Done.

**Pull request merged**